Kernel/RobSubstitution.cpp

```diff
@@ -261,7 +261,7 @@ bool RobSubstitution::occurs(VarSpec vs, TermSpec ts)
       return false;
     }
   }
-  typedef DHSet<VarSpec, VarSpec::Hash1> EncounterStore;
+  typedef DHSet<VarSpec, VarSpec::Hash1, VarSpec::Hash2> EncounterStore;
```

Comment:
So `VarSpec` has `Hash2`, but we didn't use it here...not sure why, but this seems like the intended behaviour.